elastic/integrations
# Apache: refactor dashboards

## What does this PR do?



This PR refactors dashboards for the Apache integration. Changes:
- [x] Add visualization for Connection stats: One for (apache.status.connections.async.writing, `apache.status.connections.async.keep_alive`, `apache.status.connections.async.closing`); one for (`apache.status.connections.total`)
- [x] Convert uptime information to human readable time.
- [x] Add a filter dropdown for `apache.status.hostname`
- [x] Add visualizations for `apache.status.requests_per_sec`, `apache.status.bytes_per_sec`.
- [x] Convert `apache.status.total_kbytes` to human readable representation depending on the volume of data, i.e. Mb, Gb, etc.
- [x] Convert total number of access requests to human readable representation (hundreds, thousands, millions, etc).


## Checklist



## How to test this PR locally



Use the howto guide to boot up Apache: https://github.com/elastic/elastic-package/blob/master/docs/howto/connect_service_with_elastic_stack.md
Verify if data appears on dashboards.

## Related issues


- https://github.com/elastic/integrations/issues/370

## Screenshots



<img width="1671" alt="Zrzut ekranu 2020-12-9 o 17 27 31" src="https://user-images.githubusercontent.com/14044910/101657554-5305ac80-3a44-11eb-85c5-c174fcb47cf1.png">

![screencapture-localhost-5601-app-dashboards-2020-12-09-17_31_29](https://user-images.githubusercontent.com/14044910/101657653-6ca6f400-3a44-11eb-8afb-e555c7b23d9c.png)
